monero-outreach-docs/translations/pt-br/merchants/how_to_accept_monero_pt-br.md

```diff
@@ -0,0 +1,86 @@
+# Como aceitar Monero
+
+Se você consegue instalar um aplicativo, você estará pronto para aceitar Monero (XMR) no seu negócio. Esse guia explica as formas mais comuns que comerciantes usam para aceitar Monero e como começar sem dor de cabeça a se desenvolver e a crescer o seu negócio.
+
+Primeiramente não há um tipo especial de conta comercial Monero - você apenas precisa de uma carteira e um endereço normal para começar. É recomendado que você crie um endereço separado para transações comerciais pela mesma razão de que é cômodo e seguro ter contas pessoais e de negócios separadas. Você pode usar o mesmo software para a carteira, mas ter endereços dedicados para negócios irá ajudar manter as contas organizadas.
+
+Se você ainda não tem uma carteira/endereço Monero então é melhor criar uma primeiro. As carteiras Monero GUI/CLI oficiais de [getmonero.org/downloads](https://www.getmonero.org/downloads/) são sempre recomendadas. Além disso, esse guia irá indicar carteiras mobile e você pode achar informação sobre elas na página de [Configuração da Carteira Monero](https://www.monerooutreach.org/stories/monero-wallet-quickstart.html). 
```

Comment:
For getmonero links, please check the Portuguese version link in the website, e.g. https://www.getmonero.org/pt-br/downloads/index.html. It has the pt-br language. :)